noxfile.py

```diff
@@ -145,6 +143,9 @@ def coverage_release(session: nox.Session):
         "git+https://github.com/ansible-collections/community.general",
         env={"ANSIBLE_COLLECTIONS_PATH": str(collections), **session.env},
     )
+    tmp_env = {}
+    if os.environ.get("COVERAGE_RELEASE_FORCE_TMPDIR"):
+        tmp_env["TMPDIR"] = os.environ["COVERAGE_RELEASE_FORCE_TMPDIR"]
```

Comment:
This seems redundant if `COVERAGE_RELEASE_FORCE_TMPDIR=/tmp` that's set in CI is already the default. Also, probably better not to use the `COVERAGE_` namespace if the coverage program is not actually reading this envvar.